### PR TITLE
Moved blankWorkHistoryEntry to a function

### LINKED ISSUE #303: Move blankWorkHistoryEntry into a function in util/util.js so we can test it

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #303

### CODE REVIEW

File: static/js/components/EmploymentTab.js [hunk 1]
```diff
@@ -10,23 +10,11 @@ import _ from 'lodash';
...
-      position: null,
-      industry: null,
-      company_name: null,
-      start_date: null,
-      end_date: null,
-      city: null,
-      country: null,
-      state_or_territory: null,
-    };
-  }
```
Comment (aliceriot) [inline static/js/components/EmploymentTab.js]: hooray for not needing a constructor anymore!

File: static/js/components/EmploymentTab.js [hunk 2]
```diff
@@ -103,8 +91,10 @@ class EmploymentTab extends ProfileTab {
       setWorkDialogIndex,
       setWorkDialogVisibility,
     } = this.props;
-    let clone = _.cloneDeep(profile);
-    clone['work_history'] = clone['work_history'].concat(this.blankWorkHistoryEntry);
+    let blankWorkHistoryEntry = generateNewWorkHistory();
+    let clone = Object.assign({}, profile, {
+      work_history: profile.work_history.concat(blankWorkHistoryEntry)
+    });
```
Comment (aliceriot) [inline static/js/components/EmploymentTab.js]: is the `let blankWorkHistoryEntry` line just for clarity? Wondering why not just:

```js
let clone = Object.assign({}, profile, {
  work_history: profile.work_history.concat(generateNewWorkHistory())
});
```

**Comment by aliceriot** [comment]
one question, this is great though! straightforward change.

**Comment by noisecapella** [comment]
Comment addressed

**Comment by aliceriot** [comment]
assuming tests pass

:+1: :100: :shipit:

**Pull request merged**